package.json

```diff
@@ -819,6 +819,25 @@
           "default": false,
           "description": "%config.editor_prefer_editor%"
         },
+        "i18n-ally.keygenStrategy": {
+          "type": "string",
+          "default": "default",
+          "enum": [
+            "default",
+            "random"
+          ],
+          "description": "%config.keygen_strategy%"
+        },
+        "i18n-ally.keyPrefix": {
+          "type": "string",
+          "default": "",
+          "description": "%config.key_prefix%"
+        },
```

Comment:
For the config field, I would prefer adding an `extract` scope since we are having too many configs now.

Meaning:
```jsonc
"i18n-ally.extract.keygenStrategy": "random",
"i18n-ally.extract.keyPrefix": "foo",
```

Reply:
ok